src/wp-includes/blocks.php

```diff
@@ -153,8 +153,10 @@ function register_block_style_handle( $metadata, $field_name ) {
 
 	$style_handle = generate_block_asset_handle( $metadata['name'], $field_name );
 	$block_dir    = dirname( $metadata['file'] );
```

Comment:
```suggestion
	$style_handle   = generate_block_asset_handle( $metadata['name'], $field_name );
	$block_dir      = dirname( $metadata['file'] );
```
Aligning this block of variables with the ones below (phpcs will thank you).